zoos/zoos-router.js

```diff
@@ -0,0 +1,111 @@
+const knex = require("knex");
+const router = require("express").Router();
+
+const knexConfig = {
+  client: "sqlite3",
+  connection: {
+    filename: "./data/lambda.sqlite3" //from the root folder
+  },
+  useNullAsDefault: true
+  // debug: true
+};
+const db = knex(knexConfig);
+
+// endpoints here
+
+// POST
+router.post("/", (req, res) => {
```

Comment:
great job on your endpoint logic: your code logic is great and HTTP codes are good.